pbs_light.go

```diff
@@ -91,6 +92,15 @@ type bidResult struct {
 	bid_list pbs.PBSBidSlice
 }
 
+const DEFAULT_PRICE_GRANULARITY = "med"
```

Comment:
I think if you add to `cache/cache.go` you can add the config objects and the server-side here, and then update the various cache implementations. We might also want to make this configurable from the request, right?